index.html

```diff
@@ -0,0 +1,128 @@
+      <div class="brand">
+        <img src="./img/menu.png" alt="menu-icon" />
+        <span>Video App</span>
+      </div>
+
+      <div class="input-field">
+        <input class="searchBar" type="text" />
+        <img src="./img/search.png" alt="search-icon" />
+      </div>
+
+      <div class="menu-section">
+        <ul class="top-menu">
+
+          <li><a href=""><img src="./img/notification.png" alt="" /></a></li>
+          <li><a href=""><img src="./img/video-camera.png" alt="" /></a></li>
+          <li><a href=""><img class="avatar" src="https://source.unsplash.com/random" alt="" /></a>
+          </li>
+
+        </ul>
+
+      </div>
+    </nav>
+
+    <section class="main-section">
+
+      <div class="r-side">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/oGYv3sCuxVQ" title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen></iframe>
+
+        <div class="title">
+          <h3>What does Earth sound like from space?</h3>
+        </div>
+
+          <ul class="icon-bar">
+
+            <li class="icon-cols">Like</li>
+            <li class="icon-cols">Dislike</li>
+            <li class="icon-cols">Save</li>
+            <li class="icon-cols">Share</li>
+            <li class="icon-cols"> More</li>
+
+          </ul>   
+
+        <div class="description">
+          <div class="desc-header">
+
+            <div class="user-info">
+
+              <div class="user-bio">
+                <img src="https://source.unsplash.com/random" alt="user-avatar" />
+                <span class="user-name">User1</span>
+              </div>
+
+              <small>over 10,000 views</small>
+
+              <div class="desc-body">
+
+                <p>They say space is silent, but turn your ears into radio wave receivers and suddenly it is a symphony
+                  of sounds. So what does Earth sound like from space? https://brilliant.org/astrum/ </p>
+
+              </div>
+
+            </div>
+
+          </div>
+        </div>
+
+      </div>
+
+      <div class="l-side">
+
+        <div class="pills">
+          <p class="pills-col">all</p>
+          <p class="pills-col">related</p>
+          <p class="pills-col">from channel</p>
+        </div>
+
+        <div class="l-cols">
+          <iframe width="560" height="315" src="https://www.youtube.com/embed/bW-U3BSqzFQ?controls=0"
+            title="YouTube video player" frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowfullscreen></iframe>
+
+          <div class="title">
+            <h3>Questionable objects found by NASA on Mars</h3>
+          </div>
+
+        </div>
+
+        <div class="l-cols">
+          <iframe width="560" height="315" src="https://www.youtube.com/embed/OU58Ya2-rpY?controls=0"
```

Comment:
Nice job done so far. I think it will be better to use clearer classnames (eg left-columns and left-side) than l-cols and l-side as it will enhance your codes readability.